src/shim/ShimUtils.ts

```diff
@@ -123,6 +123,19 @@ export function validateEncryptedDocument(documentData: Uint8Array) {
     }
 }
 
+/**
+ * Validate that the provided encrypted document is in the proper form, which should be a Uint8Array and be non empty.
+ */
+export function validateEncryptedDeks(edeks: Uint8Array) {
+    if (!(edeks instanceof Uint8Array)) {
+        throw new Error("Invalid encrypted deks. Edeks should be a Uint8Array.");
+    }
+
+    if (edeks.byteLength <= 0) {
+        throw new Error(`Invalid encrypted deks. Length does not meet minimum requirements.`);
+    }
+}
```

Comment:
Unit tests for this method.

Reply:
Oh, I see you're basically testing this via the `decryptUnmanaged` calls below, so that's probably fine.